src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -140,12 +157,37 @@ export class CreateClusterComponent implements OnDestroy {
           }
         });
     }
+
+    if (this.deploymentOption) {
+      const trackingId = `${this.deploymentOption} optmized deployment`;
```

Comment:
We should add a spell checker to catch these ones:
```suggestion
      const trackingId = `${this.deploymentOption} optimized deployment`;
```

Reply:
Recently Travis mentioned about the one used on the rook github workflows. We can take leverage of that probably.

Reply:
That would be nice!